content/en/hosting/analytics/setup-docker-compose.md

```diff
@@ -30,36 +30,121 @@ Configure the `COUCHDB_*` environment variables to connect to your CouchDB insta
 The first time you run the commands from any of the sections below it will need to download many Docker images and will take a while. You'll know it's done when you see `#8 DONE 0.0s` and you are returned to the command line. Be patient!
 {{% /alert %}}
 
-### Separate CouchDB instance 
+### Profiles
+
+Running CHT Sync uses docker compose profiles to organize different sets of services. 
+
+The following profiles are available:
+- `local` - for local model development
+- `production` - recommended setup for production
+- `test` - for automated tests
+
+#### Local
 
 This setup involves starting couch2pg, PostgreSQL, pgAdmin and dbt. It assumes you have a CouchDB instance running, and you updated the `.env` CouchDB variables accordingly.
 
 Run the Docker containers locally and wait for every container to be up and running:
 ```sh
-docker compose -f docker-compose.postgres.yml -f docker-compose.yml up -d
+docker compose --profile local up -d
 ```
 
-You can verify this command worked by running `docker ps`. It should show 4 containers running including couch2pg, dbt, PostgreSQL, and pgAdmin.
+You can verify this command worked by running `docker ps`. It should show four containers running including couch2pg, dbt, PostgreSQL, and pgAdmin.
```

Comment:
``````suggestion
You can verify this command worked by running `docker compose --profile local ps --format "{{.Names}}\t{{.Status}}"`. It should show four containers running including couch2pg, dbt, PostgreSQL, and pgAdmin:

```
cht-sync-couch2pg-1     Up About a minute
cht-sync-dbt-local-1    Up About a minute
cht-sync-pgadmin-1      Up About a minute
cht-sync-postgres-1     Up About a minute
```
``````